src/pages/Governance/ProposalList/GovernanceProposal/index.tsx

```diff
@@ -149,6 +151,36 @@ const GovernanceProposalUi: React.FC<GovernanceProposalProps> = ({
     abstainedVotesWei || 0,
   ]);
 
+  const statusDate = useMemo(() => {
+    switch (proposalState) {
+      case 'Active':
+        return endDate;
+      case 'Canceled':
+        return cancelDate;
+      case 'Executed':
+        return executedDate;
+      case 'Queued':
+        return queuedDate;
+      default:
+        return undefined;
+    }
+  }, [proposalState]);
+
+  const statusKey = useMemo(() => {
+    switch (proposalState) {
+      case 'Active':
+        return 'voteProposalUi.activeUntilDate';
+      case 'Canceled':
+        return 'voteProposalUi.canceledDate';
+      case 'Executed':
+        return 'voteProposalUi.executedDate';
+      case 'Queued':
+        return 'voteProposalUi.queuedUntilDate';
```

Comment:
Let's add the "Defeated" case:
<img width="873" alt="Screen Shot 2023-03-16 at 10 30 13" src="https://user-images.githubusercontent.com/44675210/225575432-1deffac7-4114-4a8f-a0b9-f13179f7236f.png">

Reply:
Added it along with the corresponding copy and `endDate`